packages/react-components/src/types/Attachment.ts

```diff
@@ -91,14 +91,14 @@ export interface AttachmentUploadOptions {
    * Accepts any type of file if not specified.
    * @beta
    */
-  acceptedMediaTypes?: string[];
+  allowedMediaTypes?: string[];
   /**
    * Allows multiple files to be selected if set to `true`.
    * Similar to the `multiple` attribute of the `<input type="file" />` element.
    * @defaultValue false
    * @beta
    */
-  canUploadMultiple?: boolean;
+  allowUploadMultiple?: boolean;
```

Comment:
Review had this to disableMultipleUploads or disableMultipleSelection as the default (undefined) is to allow multiple selection in the file input dialog